website/docs/docs/on-premises/installation/migrating-from-pact-broker.md

```diff
@@ -0,0 +1,34 @@
+---
+id: migrating
+title: Migrating from Pact Broker
+---
+
+# Migrating from Pact Broker
+
+It is possible to migrate from using the open source Pact Broker to Pactflow, keeping all existing data and making it available in Pactflow. The migration process uses the existing Pact Broker database, database migrations are then performed to add additional tables and data used for Pactflow features.
+
+If there is no existing Pact Broker instance continue to the [Database](https://docs.pactflow.io/docs/on-premises/database) section.
+
+## Single Pact Broker instance
+
+Follow existing [set up documentation](https://docs.pactflow.io/docs/on-premises) for setting up your new Pactflow, until the section titled ['Database'](https://docs.pactflow.io/docs/on-premises/database).
+
+To migrate your existing Pact Broker follow these steps instead of creating a new database:
+
+1. Re-use your existing Pact Broker environment variables, updating the names to read `PACTFLOW_` instead of `PACT_BROKER_`. It is crucial that the values for `DATABASE` related environment variables remain the same so that your Pact Broker database will be updated and used. A full list of the Pactflow environment variables can be found [here](https://docs.pactflow.io/docs/on-premises/environment-variables).
```

Comment:
Is it worth calling out the pattern for the env vars `*_DATABASE_*`

I assume a user needs to both

rename all 

`PACT_BROKER_DATABASE_*

to 

`PACTFLOW_DATABASE_*

I would maybe be use italicising or use a caution note feature of Docusaurus. [reference](https://docusaurus.io/docs/next/markdown-features/admonitions)

> existing Pact Broker database will be used, and updated for use with Pactflow as part of the process.

Just some copy change to make it clear we are updating the users existing database.